extensions/job_management/standard/requirements/job-management/update/REQ_response-locked.adoc

```diff
@@ -0,0 +1,9 @@
+[[req_job-management_update_response-locked]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/update-response-locked
+part:: If a job is locked, meaning that it is currently being processed (status set to `accepted` or `running`), the server SHALL respond with HTTP status code `423 Locked`.
+part:: The response body SHALL be based upon the OpenAPI 3.0 schema https://raw.githubusercontent.com/opengeospatial/ogcapi-processes/master/core/openapi/schemas/exception.yaml[exception.yaml].
+part:: The `type` of the exception SHALL be “http://www.opengis.net/def/exceptions/ogcapi-processes-4/1.0/locked”.
```

Comment:
What does this mean?

Reply:
Not sure about the question here?
The reported exception `type` (https://github.com/opengeospatial/ogcapi-processes/blob/master/openapi/schemas/common-core/exception.yaml#L7) must be the `http://www.opengis.net/def/exceptions/ogcapi-processes-4/1.0/locked`.

Reply:
Oh, I guess we have different error handling to consider, too.